lib/project_types/script/layers/infrastructure/script_service.rb

```diff
@@ -34,11 +34,13 @@ def set_app_script(
             configurationUi: script_config.configuration_ui,
             configurationDefinition: script_config.configuration&.to_json,
             moduleUploadUrl: module_upload_url,
-            library: {
-              language: library[:language],
-              version: library[:version],
-            },
           }
+
+          if library
+            variables[:language] = library[:language]
+            variables[:version] = library[:version]
+          end
```

Comment:
oh sorry I missed the fact that these were wrapped in a `library` object before. I think this is what we want now?

```suggestion
          variables[:library] = {
            language: library[:language],
            version: library[:version],
          }  if library
```